routes/models/och_log_entry_schema.js

```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'mongoose'.